system-addon/content-src/components/Card/Card.jsx

```diff
@@ -31,7 +31,10 @@ class Card extends React.Component {
   }
   onLinkClick(event) {
     event.preventDefault();
-    this.props.dispatch(ac.SendToMain({type: at.OPEN_LINK, data: this.props.link}));
+    this.props.dispatch(ac.SendToMain({
+      type: at.OPEN_LINK,
+      data: Object.assign(this.props.link, {where: (event.ctrlKey || event.metaKey || event.button === 1) ? "tab" : "current"})
```

Comment:
So this supports tabs but not the other click behaviors, e.g., "window" (on shift) and "save" (on alt) or other behaviors supported in `whereToOpenLink`:
https://searchfox.org/mozilla-central/source/browser/base/content/utilityOverlay.js#111-159

Not sure if there's a good way to just send the event over to call that method on the browser.